### ISSUE TITLE
Subtopics for machine learning

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/training-material#2569]
Hey @anuprulez I noticed there are quite some tutorials in https://training.galaxyproject.org/training-material/topics/statistics/

What would you think of splitting these up into subtopics like we're doing in the #2567 dev tutorials to make it look a bit more organised and provide some sub topics to help people figure out what they want.

**Comment by anuprulez** [comment]
Yes, we can do it and I will work on it. Thanks for the nice suggestion!!

**Comment by hexylena** [comment]
Oh, perfect, thanks @anuprulez !
